engine/runtime/src/test/scala/org/enso/compiler/test/core/CorePrimTest.scala

```diff
@@ -15,6 +17,7 @@ import org.scalatest.matchers.should.Matchers
   * PLEASE NOTE: Many of these tests will be removed once the smart constructors
   * exist.
   */
+@nowarn("cat=deprecation")
```

Comment:
How big is the deprecation issue? Can we not easily fix it?

Reply:
Currently major. Library doesn't export an alternative syntax.